app/containers/HomeContainer/reducer.js

```diff
@@ -11,9 +11,9 @@ export const { Types: homeContainerTypes, Creators: homeContainerCreators } = cr
   requestGetGithubRepos: ['repoName'],
   successGetGithubRepos: ['data'],
   failureGetGithubRepos: ['error'],
-  clearGithubRepos: []
+  clearGithubRepos: {}
```

Comment:
why is this an object?

Reply:
Because we get the output of api search in object not array, and this was actually making an error on console, when we considered it as an array, but on real time the result of api search was nested object, so i changed reposData and clearGithubRepos into obj

Reply:
`clearGithubRepos` is an action, it should always have an array